pulsar-flink-connector/src/main/java/org/apache/flink/streaming/util/serialization/PulsarDeserializationSchemaBuilder.java

```diff
@@ -60,9 +58,6 @@ public PulsarDeserializationSchemaBuilder() {
     }
 
     public PulsarDeserializationSchema<V> build() {
-        if (dataType == null) {
-            dataType = DataTypes.RAW(TypeInformationUtils.getTypesAsRow(recordClass)).bridgedTo(recordClass);
-        }
-        return new PulsarDeserializationSchemaWrapper<>(valueDeserializer, dataType);
+        throw new UnsupportedOperationException("PulsarDeserializationSchemaBuilder is deprecated, use PulsarDeserializationSchema#valueOnly.");
```

Comment:
Should we return the new `PulsarDeserializationSchema` instead.

Reply:
This class was marked as `deprecated` in 1.12. In upgrade 1.13, re-implementing it also needs to re-study how to implement it. Or we remove it in 1.13.